### ISSUE TITLE
MacOs High Sierra compatibility

### ISSUE THREAD

**Issue by Duffledorf** [Ascoware/get-iplayer-automator#439]
Can you confirm last compatible version for mac os 10.13.6 is "Get.iPlayer.Automator.v1.24.4.b20221118002 " and if so should there be a opsys compatibility check before installing later versions.

**Comment by skovatch** [comment]
Yes, sorry -- I mentioned that I would be dropping support for macOS 10.x in the release notes of a previous version, mainly because Xcode no longer supports building for versions prior to 10.13. I will add the minimum OS version to the updater so you won't see newer versions.

**Comment by Duffledorf** [comment]
Thanks Scott, your work is much appreciated.

**Comment by crgmrgn** [comment]
Many thanks ...

**Comment by gpk0** [comment]
Thanks all for your work here.... can anyone point me to a way reinstalling a working version now it's been overwritten by the new non-compatible one? Thanks :)

**Comment by skovatch** [comment]
https://github.com/Ascoware/get-iplayer-automator/releases will take you to all of the released versions.

**Comment by gpk0** [comment]
Great many thanks again

**Comment by MacMuser** [comment]
> Yes, sorry -- I mentioned that I would be dropping support for macOS 10.x in the release notes of a previous version, mainly because Xcode no longer supports building for versions prior to 10.13. I will add the minimum OS version to the updater so you won't see newer versions.

> Can you confirm last compatible version for mac os 10.13.6 is "Get.iPlayer.Automator.v1.24.4.b20221118002 " and if so should there be a opsys compatibility check before installing later versions.

I'm stuck on Monterey 12.6.2. I updated iPlayer-A from 1.24.4 and regretted it, lucky I have Time Machine back-ups to go back to.  I understand you need to keep software moving on but it's going to be so expensive to update a whole Mac for just one application, especially one I use every day along with a hacked iTunes install that's no longer available. I fill my old iPod with BBC daily radio shows to listen to all night. I shall be completely stuck without that.

**Comment by skovatch** [comment]
You'll be fine on Monterey, which is 12.x. It will be a long time before I do anything that requires macOS Ventura.

**Comment by MacMuser** [comment]
Phew! Thanks
